pwa/src/components/DiagnosisFields.json

```diff
@@ -0,0 +1,196 @@
+{
+    "diagnoses": [
+        {
+            "name": "Asthma",
+            "id": "asthma"
+        },
+        {
+            "name": "Allergies",
+            "id": "allergies"
+        },
+        {
+            "name": "Anemia",
+            "id": "anemia"
+        },
+        {
+            "name": "Malnutrition",
+            "id": "malnutrition"
+        },
+        {
+            "name": "Depression",
+            "id": "depression"
+        },
+        {
+            "name": "Diabetes",
+            "id": "diabetes"
+        },
+        {
+            "id": "eye",
+            "group-title": "Eye",
+            "group": [
+                {
+                    "name": "Ptergium",
+                    "id": "ptergium"
+                },
+                {
+                    "name": "Cataract",
+                    "id": "cataract"
+                },
+                {
+                    "name": "Dry Eye",
+                    "id": "dry-eye"
+                },
+                {
+                    "name": "Conjuntivitis",
+                    "id": "conjuntivitis"
+                }
+            ]
+        },
+        {
+            "name": "Fever",
+            "id": "fever"
+        },
+        {
+            "name": "Possible Malaria",
+            "id": "malaria"
+        },
+        {
+            "name": "Filariasis",
+            "id": "filariasis"
+        },
+        {
+            "id": "intestinal",
+            "group-title": "Intestinal",
+            "group": [
+                { "name": "Gastritis", "id": "gastritis" },
+                { "name": "GERD", "id": "gerd" },
+                { "name": "Gastroenteritis", "id": "gastroenteritis" },
+                { "name": "Dysentery", "id": "dysentery" },
+                { "name": "Possible Cholera", "id": "cholera" }
+            ]
+        },
+        {
+            "name": "Headache",
+            "id": "headache"
+        },
+        {
+            "group-title": "Hernia",
+            "id": "hernia",
+            "group": [
+                { "name": "Inguinal", "id": "inguinal" },
+                { "name": "Umbilical", "id": "umbilical" }
+            ]
+        },
+        {
+            "id": "cardiovascular",
+            "group-title": "Cardiovascular",
+            "group": [
+                { "name": "Hypertension", "id": "hypertension" },
+                { "name": "Stroke", "id": "stroke" },
+                { "name": "Angina", "id": "angina" },
+                { "name": "CHF", "id": "chf" }
+            ]
+        },
+        {
+            "name": "Healthy",
+            "id": "healthy"
+        },
+
+        {
+            "name": "Musculoskeletal Pain",
+            "id": "musculoskeletal"
+        },
+        {
+            "group-title": "Otitis",
+            "id": "otitis",
+            "group": [
+                { "name": "Right", "id": "otitis-right" },
+                { "name": "Left", "id": "otitis-left" }
```

Comment:
Isn't there an Otitis B on the green card as well?

Reply:
Good catch - That's my fault, I assumed it meant right/left/both (which I guess it technically does). It was significantly easier in the case of using the json/dynamic component creation to make a checkbox than to use a radio button because it requires using state. 
I had imagined that instead of selecting "Bilateral" they could select both "Left" and "Right". I think this is a decision that could use other team member's feedback.

This is also the case for Pneumonia and Sinusitis.

Reply:
I double checked the db schema and it looks like only Otitis has a specification for which side. I'll remove the options for Pneumonia and Sinusitis. We should still discuss (maybe briefly after standup today) whether we want L/R/B radio buttons or L/R checkboxes.

Reply:
Sinusitis and Pneumonia L/R removed in [this commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/84/commits/bcb6367f3ab4bdd1cb4089f51311a383e696553d)